content/blog/sivabala-opsql-opsramps-kalpavriksha-wish-fulfilling-tree.md

```diff
@@ -0,0 +1,194 @@
+---
+title: "OpsRamp OpsQL API:  IT Operations kalpavriksha(wish-fulfilling tree)"
```

Comment:
Hi 
Reading through the blog post, I personally do not understand the title "kalpavriksha(wish-fulfilling tree)".
In the introductory paragraph you might want to add a brief description about what does "kalpavriksha(wish-fulfilling tree)" mean in the context of OpsRamp QL.